### PR TITLE
Items endpoint

### PR DESCRIPTION
**Pull request by OKrivtsova**
tests for /items endpoint
+ some fixing of folder structure

### CODE REVIEW

**Comment by AntonShopin** [comment]
Could you please add tests for the following includes ('items' endpoint):
* product-measurement-units
* vouchers (if possible)
* bundled-items  (not 'bundle-items')
* abstract-products

File: tests/api/b2b/glue/cart_endpoints/items/negative.robot [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+*** Test Cases ***
+####### POST #######
+Add_item_to_cart_non_existing_sku
+    [Setup]    Run Keywords    I get access token for the customer:    ${yves_user_email}
+    ...    AND    I set Headers:    Content-Type=${default_header_content_type}    Authorization=${token}
+    ...    AND    I send a POST request:    /carts    {"data":{"type":"carts","attributes":{"priceMode":"${gross_mode}","currency":"${currency_code_eur}","store":"${store_de}","name":"Cart-${random}"}}}
+    ...    AND    Save value to a variable:    [data][id]    cart_uid
+    When I send a POST request:    /carts/${cart_uid}/items    {"data": {"type": "items","attributes": {"sku": "fake","quantity": 1}}}
+    Then Response status code should be:    422
+    And Response reason should be:    Unprocessable Entity
```
Comment (AntonShopin) [inline tests/api/b2b/glue/cart_endpoints/items/negative.robot:14]: tests are failing on this assertion. Is it ok?
"Unprocessable Entity != Unprocessable Content"

Comment (OKrivtsova) [reply tests/api/b2b/glue/cart_endpoints/items/negative.robot:14]: they are not failing for me
image

**Comment by OKrivtsova** [comment]
> 
@AntonShopin 
most of the includes are not direct includes - e.g. abstracts are included into concretes concretes into items and items only are included into the actual request. I added them but see no reason to test that - this way we in every test may test ALL entities in spryker as includes because they are related via 2nd and 3rd relation... we need to keep tests as insular as possible.
Voucher includes should be tested in voucher endpoint.

**Review by AntonShopin — APPROVED** [review]

**Pull request merged**